src/shared/components/workbasket-filters/workbasket-filters.component.html

```diff
@@ -25,7 +25,7 @@ <h2 class="heading-h2" aria-label="Filters">Filters</h2>
     <select class="form-control form-control-3-4 ccd-dropdown" id="wb-case-state"
             name="state" [(ngModel)]="selected.caseState" [disabled]="isCaseStatesDropdownDisabled()"
             aria-controls="search-result">
-      <option value="">Any</option>
+      <!-- <option *ngIf="!workbasketDefaults" [ngValue]="null">Any</option> -->
```

Comment:
if this is not needed anymore just delete it rather than comment out

Reply:
understood

Reply:
deleted